outputs/output_0.csv

```diff
@@ -0,0 +1,7 @@
+year,G,N,R,R/G
```

Comment:
I think we should remove the generated outputs from this PR and add the outputs folder to the `.gitignore` so we all don't end up accidentally committing them when we run our local tests.